kwalitee/release.py

```diff
@@ -0,0 +1,40 @@
+import json, re
+import requests
+
+class Release():
+
+    def __init__(self, json):
+        self.tag = json['tag_name']
+        self.projects = self.__get_projects(json)
+
+    def __str__(self):
+        return f'<Release {self.tag}>'
+
+    def to_json(self):
+        desc = {}
+        desc['tag'] = self.tag
+        desc['projects'] = self.projects
+        return desc
+
+    def __get_projects(self, json_data):
+        markdown_desc = json_data['body']
+        pattern = r'### \[(.*?)\]'
+        return re.findall(pattern, markdown_desc)
+
+
+def get_releases(test_mode=False):
+    if test_mode:
+        f = open('tests/assets/ocrd_all_releases.json')
+        response_json = json.load(f)
+    else:
+        api_url = "https://api.github.com/repos/OCR-D/ocrd_all/releases"
```

Comment:
If it's really just about getting the release notes, this seems unnecessary. ocrd_all is checked out and the release notes only consist of the newest section in the `CHANGELOG.md`. So you could get that information from the local ocrd_all clone. But I think the better approach is to inspect with git (see above).